src/frontend/parallelization_phase.nit

```diff
@@ -39,10 +39,9 @@ private class ParallelizationPhase
 			toolcontext.error(nmethdef.location, "Syntax error: only a method can be threaded.")
 			return
 		end
-		if nmethdef.n_signature.n_params.length != 0 then
-			toolcontext.error(nmethdef.location, "Syntax error: parametrized method not supported yet.")
-			return
-		end
+
+		#TODO: check for self calls
```

Comment:
What's the plan here for v3? I see 2 main options:

1. Forbid all calls on self.
2. Redirect all calls on self to the original `self`. This could get complicated, is it worth it?